pkg/draft/pack/load.go

```diff
@@ -14,21 +16,31 @@ import (
 // and hand off to the appropriate pack reader.
 func FromDir(dir string) (*Pack, error) {
 	pack := new(Pack)
+	pack.Files = make(map[string]io.ReadCloser)
 
 	topdir, err := filepath.Abs(dir)
 	if err != nil {
 		return nil, err
 	}
 
-	pack.Chart, err = chartutil.LoadDir(filepath.Join(topdir, ChartsDir))
+	files, err := ioutil.ReadDir(topdir)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error reading %s: %s", topdir, err)
 	}
-
-	dockerfile := filepath.Join(topdir, DockerfileName)
-	pack.Dockerfile, err = ioutil.ReadFile(dockerfile)
-	if err != nil {
-		return nil, fmt.Errorf("error reading %s: %s", dockerfile, err)
+	for _, fInfo := range files {
+		if fInfo.IsDir() {
+			chart, err := chartutil.LoadDir(filepath.Join(topdir, fInfo.Name()))
```

Comment:
hmm... This makes the assumption that *every* directory in the pack is a chart. Do we really want to make that assumption?

Reply:
Yeah I agree, it would be good to copy any directory rather than them having to be chart folders.  Do you have any ideas on how best to do that?

Reply:
One way is to save the extra files as a map of file names and byte arrays(`files map[string][]byte`). We can save that on the `Pack` struct and then write those files in `pack.SaveDir`. If that seems like a sane suggestion for files, then we can extend that analogy further for copying directories and instead add a map of maps to the `Pack` struct: `directories map[string]files` with the string being the name of the directory.

Another way to implement this could be to leave the FromDir as is and in SaveDir, go back to the pack directory and loop through all the directories and copy into the current working directory leaving aside the Chart directory and the Dockerfile which have already been taken care of and are essential pieces in a `pack`

What do you think?

Reply:
Thanks @michelleN yeah sounds good.  I'm a bit snowed under ATM so might wait for #453 to merge first and then this PR will be a bit cleaner if that's ok with you.